docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -204,32 +214,40 @@ However, we need to point out that, certain resources are not accessible from re
 The key changes to realize managability and automony are:
 * A new field, ApplicableRange, will be added to the CustomResourceDefinitionSpec. The field is ignored in a regular-tenant CRD. Yet for a system-tenant CRD, it indicates the scope of the regular tenants that this CRD is applicable to. This field is a list of annotations, which are used to finding matching tenants. (other designs may also work, as long as the field give info in finding matching tenants)
 
+* A system tenant CRD with the following label is forced shared, which overrides all the local CRDs of regular tenants which has the same name.
+
+  **arktos.futurewei.com/crd-sharing-policy: forced**
+ 
 * the logic to find a matching CRD for a request in the API server will be changed as the following:
 
 ```
 GetMatchingCRD(G/V/K, tenant) (found_CRD, error)
+  if (found a system forced sharing CRD) {
+    return (the system CRD), nil
+  }
```

Comment:
So "SystemForced" CRDs will be return even if the tenant sets a policy of "NeverUseSystemCRD"?

Reply:
updated.